pkg/kudoctl/resources/dependencies/resolve_test.go

```diff
@@ -31,6 +34,32 @@ func (resolver nameResolver) Resolve(
 }
 
 func createPackage(name string, dependencies ...string) packages.Package {
+	ovVersion := "0.0.1"
```

Comment:
operatorVersion? this is operatorVersionVersion :-)

Reply:
🙈 Indeed